### ISSUE TITLE
Cannot re-import a previously imported folder

### ISSUE THREAD

**Issue by olliejm** [beetbox/beets#2710]
### Problem

Running this command in verbose (`-vv`) mode:

$ beet -vv import /path/to/music

Led to this problem:

Skipping previously-imported path: /path/to/music/Some Artist/Some Album
... (repeat for all folders)
No files imported from /path/to/music
Skipped 922 paths.
Sending event: import
Sending event: cli_exit

### Setup

* OS: Raspian Stretch Lite
* Python version: 2.7.13
* beets version: 1.4.5
* Turning off plugins made problem go away (yes/no): no

My configuration (output of `beet config`) is:

```yaml
paths:
    default: $albumartist/${album}%aunique{} ($year)/$disc_and_track $title
    comp: Compilations/${album}%aunique{} ($year)/$disc_and_track $title
mpd:
    host: localhost
    port: 6600
    rating: yes
    rating_mix: 0.75
    music_directory: /path/to/music
    password: REDACTED
item_fields:
    disc_and_track: u'%02i-%02i' % (disc, track) if disctotal > 1 else u'%02i' % (track)
per_disc_numbering: yes
asciify_paths: yes
fetchart:
    minwidth: 500
    auto: yes
    sources:
    - filesystem
    - coverart
    - itunes
    - amazon
    - albumart
    google_engine: 001442825323518660753:hrh5ch1gjzm
    enforce_ratio: no
    cautious: no
    maxwidth: 0
    store_source: no
    google_key: REDACTED
    fanarttv_key: REDACTED
    cover_names:
    - cover
    - front
    - art
    - album
    - folder
library: /home/pi/.config/beets/library.db
original_date: yes
ftintitle:
    format: (feat. {0})
    auto: yes
    drop: no

plugins: inline fetchart ftintitle mpdstats mpdupdate
directory: /mnt/external/music

import:
    write: yes
    copy: no
    incremental: no

match:
    preferred:
        media: [CD, Digital Media|File]
pathfields: {}
album_fields: {}
```

Basically even though incremental is turned off and I have tried deleting .db files as well as uninstalling and reinstalling beets, it always skips all directories when trying to import my library again (I want to re-import with some different settings).

**Comment by sampsyo** [comment]
Hmm; that's odd. Is there any chance beets is looking at a different configuration file from the one you think it's looking at? You could try removing that file, for example, to see if there's any effect.

**Comment by transiti0nary** [comment]
I had the same thought, but I'm not sure where any other configuration file might be? I couldn't see one when scanning through the file system but then I don't exactly know where it would be anyway

-------- Original Message --------

**Comment by sampsyo** [comment]
Well, the `beet config` command can help—is that what you used to produce the paste above?

**Comment by transiti0nary** [comment]
Yeah beet config is what I used to get that paste

-------- Original Message --------

**Comment by sampsyo** [comment]
Huh! I’m not exactly sure how to proceed—needles to say, this isn’t happening here. Maybe it would be worth experimenting with changing your configuration around to see if there’s any effect.

**Comment by jhermann** [comment]
`state.pickle` wasn't mentioned…

**Comment by olliejm** [comment]
I'm sorry this was so long ago that I forgot exactly how it resolved - but I imagine it was also related to the state.pickle issue mentioned above. Safe to say I don't have the problem now with beets on either windows, osx or linux. So I'll close
